django/db/models/sql/query.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, model, where=WhereNode, alias_cols=True):
     @property
     def output_field(self):
         if len(self.select) == 1:
-            return self.select[0].field
+            return self.select[0].target
```

Comment:
Looks like this will need to be 
```suggestion
            select = self.select[0]
            return getattr(select, 'target', None) or  select.output_field
```